packages/lucia/src/middleware/index.ts

```diff
@@ -12,6 +12,9 @@ import type {
 	Response as ExpressResponse
 } from "express";
 
+import "@fastify/cookie";
+import { FastifyReply, FastifyRequest } from "fastify";
```

Comment:
Omitting `type` will break everything

```suggestion
import type { FastifyReply, FastifyRequest } from "fastify";
```

Reply:
Got it. Thanks!